Tools/WebKitTestRunner/TestController.h

```diff
@@ -756,6 +757,9 @@ class TestController {
     std::optional<uint64_t> m_downloadTotalBytesExpectedToWrite;
     bool m_shouldLogDownloadSize { false };
     bool m_shouldLogDownloadExpectedSize { false };
+    size_t m_downloadIndex { 0 };
+    bool m_shouldLogDownloadSize { false };
```

Comment:
This is already present at line 758. The build error is about this data member being defined twice. You need to drop this one.